docs/content/projects/cat.md

```diff
@@ -0,0 +1,46 @@
+---
+title: "OSCAL CAT"
+description: "OSCAL Catalog Authoring Tool (CAT) - import, edit, and export OSCAL Profiles from the browser"
+heading: "CAT: OSCAL Catalog Authoring Tool"
+toc:
+  enabled: true
+usabanner: true
+---
+
+## What is CAT?
+
+The OSCAL Catalog Authoring Tool (CAT) enables users to create, scope, tailor control catalogs by creating [OSCAL Profiles in JSON format](https://pages.nist.gov/OSCAL/reference/latest/profile/json-reference/).
```

Comment:
Suggest adding at the end of the sentence that these profiles can be subsequently converted into new catalogs using the profile resolution...

Reply:
Added